### PR TITLE
Update to so number 164

### LINKED ISSUE #27: @conda-forge-admin please rerender
### Additional comment:

_No response_

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #27.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

Fixes #27

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@h-vetinari is this the openssl bug that is causing win64 to fail?

**Comment by hmaarrfk** [comment]
maybe the m2 packages need updating?

**Comment by h-vetinari** [comment]
> @h-vetinari is this the openssl bug that is causing win64 to fail?

Could definitely be the case - 1.0.2g is _very_ [old](https://github.com/openssl/openssl/releases/tag/OpenSSL_1_0_2g).
 
> maybe the m2 packages need updating?

Yeah, that could be the case. https://github.com/conda-forge/msys2-recipes hasn't been updated in quite some time, and I also don't see openssl there. Not sure if it's even intended to update the m2-recipes still (CC @isuruf)?

Upstream msys2 has a current [openssl](https://packages.msys2.org/base/mingw-w64-openssl) that includes the fix for the expired certificate.

**Comment by h-vetinari** [comment]
> I also don't see openssl there

Seems that @isuruf uploaded this [manually](https://anaconda.org/conda-forge/m2-openssl/files) about 1.5 years ago. Pretty sure that there's no feedstock for this (it doesn't [appear](https://anaconda.org/conda-forge/m2-openssl/files) in feedstock-outputs at all). Perhaps needs to go through staged-recipes?

**Comment by hmaarrfk** [comment]
ok, i just manually exported the variables we needed to ensure the version is correct.

From the log:
```
2022-07-08T00:18:43.0819869Z + ./version.sh
2022-07-08T00:18:43.0835235Z #define X264_REV 3095
2022-07-08T00:18:43.0836528Z #define X264_REV_DIFF 0
2022-07-08T00:18:43.0836939Z #define X264_VERSION " r3095 baee400"
2022-07-08T00:18:43.0837360Z #define X264_POINTVER "0.164.3095 baee400"
```

locally
```
$ ./version.sh 
#define X264_REV 3095
#define X264_REV_DIFF 0
#define X264_VERSION " r3095 baee400"
#define X264_POINTVER "0.164.3095 baee400"
```

**Comment by hmaarrfk** [comment]
I'm also suggesting that:
- we rename the lib file on windows, so a review of the proposed changes would be appreciated. This should help ffmpeg find the library.
- disable static libraries on unix

Let me know what you all think.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,25 +35,21 @@ requirements:
     - perl
     - nasm
     - make
-    # m2-git is required, because "git clone" won't work when combining the "git" package with "m2-bash"
-    - {{ posix }}git  # [win]
     - {{ posix }}bash  # [win]
     - {{ posix }}coreutils  # [win]
     - {{ posix }}grep  # [win]
     - {{ posix }}sed  # [win]
-  host:
```
Comment (h-vetinari) [inline recipe/meta.yaml:37]: An empty host section was often used to enforce that recipes are detected as cb3, not sure if this is relevant here.

**Review by h-vetinari — COMMENTED** [review]
Looks good to me

**Comment by h-vetinari** [comment]
> Let me know what you all think.

Fine with rename of DLL and removing static libs on unix.

**Comment by h-vetinari** [comment]
PS. Out of curiosity, how did you work around the ancient m2-openssl?

**Comment by hmaarrfk** [comment]
> PS. Out of curiosity, how did you work around the ancient m2-openssl?

i stopped using git. and rewrote the version.sh file using the conda recipe.

**Pull request merged**